tests/modules/images/webp-uploads/helper-tests.php

```diff
@@ -364,6 +364,7 @@ public function it_should_return_empty_array_when_filter_returns_empty_array() {
 	 * @test
 	 */
 	public function it_should_return_default_transforms_when_filter_returns_non_array_type() {
+		/** @phpstan-ignore-next-line */
 		add_filter( 'webp_uploads_upload_image_mime_transforms', '__return_null' );
```

Comment:
Strange. This is reported as an error for me locally but it isn't in GHA?

Reply:
It's also an error being reported in #898, but not in trunk.

Reply:
Strange issue 🤷🏻‍♂️

Reply:
What's the error? Is it maybe because of a different PHPStan strictness or other config? Or because of a different PHP version? 🤔

Reply:
Looks like a `composer update` fixed it!